rllib/policy/eager_tf_policy.py

```diff
@@ -325,16 +325,15 @@ def compute_actions(self,
             self._is_training = False
             self._state_in = state_batches
 
-            if tf.executing_eagerly():
-                n = len(obs_batch)
-            else:
-                n = obs_batch.shape[0]
-            seq_lens = tf.ones(n, dtype=tf.int32)
+            if not tf1.executing_eagerly():
+                tf1.enable_eager_execution()
```

Comment:
I don't think we should ever be trying to enable eager in the middle of any call, only at setup. Note that eager execution might return False in trace mode.
Why not keep it as before?

Reply:
The situation before was even worse, I think. We were basically running an eager_tf_policy, w/o being in eager mode. That's why we had to check with the previous if-block.